src/griffe/cli.py

```diff
@@ -24,6 +24,7 @@
 
 import colorama
 
+from griffe import __version__
```

Comment:
If we use importlib instead:

```python
from importlib.metadata import version as get_version

version = get_version("griffe")
```

Reply:
Hi,

Thanks for the review. Sounds good to me.I added this suggestion.